### PR TITLE
Clean up MVKLogging

### PR DESCRIPTION
**Pull request by cfnptr**
* Replaced ASL levels with MVKConfigLogLevel.
* Moved ```MVKLogging.h``` to ```Utility/``` directory.
* Added MVKConfigLogLevel Warning and Debug levels.

### CODE REVIEW

**Comment by cfnptr** [comment]
#1457

File: MoltenVK/MoltenVK/Utility/MVKBaseObject.h [hunk 1]
```diff
@@ -19,6 +19,7 @@
 #pragma once
 
 #include "mvk_vulkan.h"
```
Comment (billhollings) [inline MoltenVK/MoltenVK/Utility/MVKBaseObject.h:21]: Can remove `mvk_vulkan.h` now, as it's already included in `vk_mvk_moltenvk.h`.

File: MoltenVK/MoltenVK/Utility/MVKLogging.h [hunk 2]
```diff
@@ -28,16 +28,15 @@ extern "C" {
...
 
 /**
  * This library adds flexible, non-intrusive logging and assertion capabilities
  * that can be efficiently enabled or disabled via compiler switches.
  *
  * There are five levels of logging: Trace, Info, Warning, Error and Debug, and each can be enabled
- * independently via the MVK_LOG_LEVEL_TRACE, MVK_LOG_LEVEL_INFO, MVK_LOG_LEVEL_WARNING,
- * MVK_LOG_LEVEL_ERROR and MVK_LOG_LEVEL_DEBUG switches, respectively.
+ * independently via the MVK_LOG_LEVEL_TRACE, MVK_LOG_LEVEL_DEBUG, MVK_LOG_LEVEL_INFO,
+ * MVK_LOG_LEVEL_WARNING анд MVK_LOG_LEVEL_ERROR switches, respectively.
```
Comment (billhollings) [inline MoltenVK/MoltenVK/Utility/MVKLogging.h:39]: Clean up the list of five logging levels and how they map to `MVK_LOG_LEVEL_*`s. Also...is that Cyrillic text (or something) in the "and"?

**Review by billhollings — CHANGES_REQUESTED** [review]
LGTM. Just a couple of non-functional documentation and header changes requested.

File: MoltenVK/MoltenVK/Utility/MVKLogging.h (hunk 2 above)
Comment (cfnptr) [reply MoltenVK/MoltenVK/Utility/MVKLogging.h:39]: > Clean up the list of five logging levels and how they map to MVK_LOG_LEVEL_*s.

What do you mean, remove these lines from the description? Or reorder ```Trace, Info, Warning, Error and Debug``` line?

> ...Also...is that Cyrillic text (or something) in the "and"?

Oh yes, I'm sorry, I didn't notice.

Comment (billhollings) [reply MoltenVK/MoltenVK/Utility/MVKLogging.h:39]: >Or reorder Trace, Info, Warning, Error and Debug line?

Yes. The "respectively" indicates the reader should expect the same ordering between the `Trace, Info...` list and and "MVK_LOG_LEVEL..." list.

Comment (billhollings) [reply MoltenVK/MoltenVK/Utility/MVKLogging.h:39]: But frankly...I'm also fine if you remove the first list, and just have:

```
There are five levels of logging, controlled by the MVK_LOG_LEVEL_TRACE, MVK_LOG_LEVEL_DEBUG,
MVK_LOG_LEVEL_INFO, MVK_LOG_LEVEL_WARNING аnd MVK_LOG_LEVEL_ERROR switches.
```

File: MoltenVK/MoltenVK/Utility/MVKBaseObject.h (hunk 1 above)
Comment (cfnptr) [reply MoltenVK/MoltenVK/Utility/MVKBaseObject.h]: Fixed. 0da221f4

File: MoltenVK/MoltenVK/Utility/MVKLogging.h (hunk 2 above)
Comment (cfnptr) [reply MoltenVK/MoltenVK/Utility/MVKLogging.h]: Fixed. 0da221f4

**Review by billhollings — APPROVED** [review]

**Pull request merged**